MatrixKit/Views/Authentication/MXKAuthInputsView.h

```diff
@@ -83,6 +83,12 @@ typedef enum {
  Tell the delegate to cancel the current operation.
  */
 - (void)authInputsViewDidCancelOperation:(MXKAuthInputsView *)authInputsView;
+
+/**
+ Tell the delegate to autodiscover the server configuration.
+ */
+- (void)authInputsView:(MXKAuthInputsView *)authInputsView autoDiscoverServerWithDomain:(NSString*)domain;
```

Comment:
Implementation in `MXKAuthInputsView.m`seems to be missing

Reply:
This is expected like for `presentViewController` in this delegate. We are in the optional part.

Reply:
My bad I thought I saw this method in the header not in the protocol. Ignore this comment :)